Makefile

```diff
@@ -94,10 +94,17 @@ ifeq (${DBUILD_SITE_URL}, )
   export DBUILD_SITE_URL
 endif
 
+# Specify the kubeconfig path to a Kubernetes cluster 
+# to run Hostpath integration tests
+ifeq (${KUBECONFIG}, )
+  KUBECONFIG=${HOME}/.kube/config
+  export KUBECONFIG
+endif
+
 export DBUILD_ARGS=--build-arg DBUILD_DATE=${DBUILD_DATE} --build-arg DBUILD_REPO_URL=${DBUILD_REPO_URL} --build-arg DBUILD_SITE_URL=${DBUILD_SITE_URL} --build-arg BRANCH=${BRANCH} --build-arg RELEASE_TAG=${RELEASE_TAG}
 
 .PHONY: all
-all: test provisioner-localpv-image
+all: test provisioner-localpv-image hostpath-integration-test
```

Comment:
Not all developer machines will have minikube or K8s installed. The integration test should be invoked expclictly.

Reply:
Removed this.